Makefile

```diff
@@ -49,7 +49,7 @@ check: flake8 test
 .PHONY: pylint
 pylint:
 	pylint -d missing-docstring *.py kiwi_lint/
-	PYTHONPATH=.:./tcms/ DJANGO_SETTINGS_MODULE=$(DJANGO_SETTINGS_MODULE) pylint --load-plugins=pylint_django --load-plugins=kiwi_lint --load-plugins=pylint.extensions.docparams -d missing-docstring -d duplicate-code tcms/ tcms_settings_dir/
+	PYTHONPATH=.:./tcms/ DJANGO_SETTINGS_MODULE=$(DJANGO_SETTINGS_MODULE) pylint --load-plugins=pylint_django --load-plugins=kiwi_lint --load-plugins=pylint.extensions.docparams --load-plugins=pylint_django.checkers.migrations -d missing-docstring -d duplicate-code tcms/ tcms_settings_dir/
```

Comment:
This form of execution generates too many unrelated errors for the migration files. Execute the migration checkers separately like so:

```
PYTHONPATH=.:./tcms/ DJANGO_SETTINGS_MODULE=tcms.settings.test pylint --load-plugins=pylint_django.checkers.migrations -d missing-docstring -d duplicate-code  tcms/*/migrations/*
```

before the main linter is executed. 

From what I can see this reports genuine issues that should be fixed, incl. some in pylint_django itself.